manager/integration/tests/test_stress.py

```diff
@@ -64,9 +69,10 @@ def get_md5sum(file_path):
 
 
 def write_data(k8s_api_client, pod_name):
-    src_dir_path = '/tmp/'
+    src_dir_path = STRESS_RANDOM_DATA_DIR
     dest_dir_path = '/data/'
-    file_name = 'data-' + pod_name + '.bin'
+    file_name = STESS_DATAFILE_NAME_PREFIX + pod_name + \
```

Comment:
We can use a e.g. `get_data_filename()` function.

Reply:
done